types/ionic-prettier-config.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@ionic/prettier-config';
```

Comment:
had to add this in order to support the small `formatCode` helper I added for fixing tests.